src/main/resources/public/ts/sniplets/content/workspace-nextcloud-content.sniplet.ts

```diff
@@ -207,11 +207,14 @@ class ViewModel implements IViewModel {
 
     private processMoveTree(folderContent: any, document: SyncDocument, selectedFolderFromNextcloudTree: SyncDocument): void {
         if (folderContent.folder instanceof models.Element) {
+            const nextcloudController: any = this.getNextcloudTreeController();
             const filesToMove: Set<SyncDocument> = new Set(this.selectedDocuments).add(document);
             const filesPath: Array<string> = Array.from(filesToMove).map((file: SyncDocument) => file.path);
             if (filesPath.length) {
                 this.nextcloudService.moveDocumentNextcloudToWorkspace(model.me.userId, filesPath, folderContent.folder._id)
-                    .then(() => {
+                    .then(() => nextcloudController.nextcloudUserService.getUserInfo(model.me.userId))
```

Comment:
I have to retrieve user infos here in order to get the updated quota (according to server). This need to be done every time a file is deleted or uploaded or move out of the nextcloud.